cogs/updates.py

```diff
@@ -1,38 +1,113 @@
+        subcmds = []
+        for cmd in ctx.command.commands:
+            try:
+                if all(check(ctx) for check in cmd.checks):
+                    subcmds.append('`'+cmd.name+'`')
+            except commands.errors.MissingPermissions as e:
+                logging.debug(f'Command {cmd.name} discarded in {ctx.command.name} group subcommand list.')
+        await ctx.send(f'Available subcommands: {", ".join(subcmds)}.')
     
     @update.command()
-    async def set(self, ctx: commands.Context):
+    @admin_command()
+    async def set(self, ctx: commands.Context, channel: discord.TextChannel):
         """
         Set an updates channel.
-        """ 
-        raise NotImplementedError('Command requires implementation and permission set-up.')
+        """
+        values = dict(updates_id = channel.id)
+        await GuildData.update_or_create(values, guild_id = ctx.guild.id)
+        await ctx.send(f'Updates channel set to <#{channel.id}>.')
     
     @update.command(aliases=['add'])
-    async def create(self, ctx: commands.Context):
+    async def create(self, ctx: commands.Context, update1: str, update2: Optional[str] = None, update3: Optional[str] = None):
         """
-        Create an update entry.
+        Create (up to 3) update entries.
         """
-        raise NotImplementedError('Command requires implementation and permission set-up.')
+        # note: currently, discord does not support variable-length arguments in hybrid commands
+        updates = [update1]
+        if update2: updates.append(update2)
+        if update3: updates.append(update3)
+
+        guild_data = await GuildData.get(guild_id = ctx.guild.id)
+        updates_channel = self.bot.get_channel(guild_data.updates_id)
+        
+        # updates operates by UTC dates
+        last_midnight = datetime.now(tz=UTC).replace(hour=0, minute=0, second=0, microsecond=0)
+        async for message in updates_channel.history(limit=5, after=last_midnight):
+            if message.author == self.bot.user:
+                update_content = '\n- '.join([message.content, *updates])
+                await message.edit(content=update_content)
+                break
+        else:
+            # if no message was found:
+            d = last_midnight.date()
+            fmt = f"**{ordinal(d.day)} of %B, %Y**"
+            update_date = d.strftime(fmt)
+            update_content = '\n- '.join([update_date, *updates])
+            await updates_channel.send(content=update_content)
+
+        await ctx.send(f'Added update(s): `{"`,`".join(updates)}`')
     
     @update.command(aliases=['remove'])
-    async def delete(self, ctx: commands.Context):
+    async def delete(self, ctx: commands.Context, message: UpdatesMessage, entries: commands.Greedy[Index]):
         """
-        Delete an update entry.
+        Delete update entries.
         """
-        raise NotImplementedError('Command requires implementation and permission set-up.')
+        print(entries)
```

Comment:
Should this be `print` or `logger.debug`?

Reply:
Was this not fixed in the second commit?